packages/ui-library/src/theme/typography.ts

```diff
@@ -143,4 +143,12 @@ export const buildTypographyVariant = (mode: PaletteMode): TypographyOptions =>
     fontWeight: 500,
     lineHeight: 1.2, // 12/10
   },
+  confirmationLoading: {
```

Comment:
Not a fan of this, we have just one use case for this. What do you think of adding this as a StyledTypography?